template/portainer-v2-arm32.json

```
@@ -5551,6 +5551,45 @@
 					"container": "/config"
 				}
 			]
+		},
+		{
+			"categories": [
+				"Other"
+			],
+			"description": "Rdesktop is an Ubuntu based container containing full desktop environments in officially supported flavors accessible via RDP.",
+			"env": [
+				{
+					"default": "1000",
+					"label": "PUID",
+					"name": "PUID"
+				},
+				{
+					"default": "1000",
+					"label": "PGID",
+					"name": "PGID"
+				},
+                {
+					"default": "America/New_York",
+                    "label": "TZ",
+                    "name": "TZ"
+                }
+			],
+			"image": "linuxserver/rdesktop:latest",
+			"logo": "https://raw.githubusercontent.com/ItsNoss/pi-hosted/master/images/rdesktop.png",
```

Point to correct repo
```suggestion
			"logo": "https://raw.githubusercontent.com/pi-hosted/pi-hosted/master/images/rdesktop.png",
```